packages/support/src/settings/_variables.color.scss

```diff
@@ -74,8 +74,14 @@ $color-base-inverse: $color-white !default;
 $color-muted-inverse: #bac5cf !default;
 
 // Focus
-$color-focus: #3e94cf !default;
-$color-focus-inverse: #59bcff !default;
+$color-focus: #ffde11 !default;
+$color-focus-inverse: #ffde11 !default;
+$focus-border-inverse: #8f7c00 !default;
+$focus-shadow: inset 0 0 0 1px $color-base;
+$focus-shadow-inverse: inset 0 0 0 1px $color-base !default;
+$focus-shadow-link: 0 3px $color-base;
+$focus-shadow-link-inverse: 0 3px $focus-border-inverse !default;
```

Comment:
I'm thinking we should use the following format for focus styles
- $focus-NAME-NAME

For example, 
- $focus-color
- $focus-color-inverse
- $focus-border-color-inverse
- $focus-shadow
- etc...

Reply:
i like this

Reply:
@line47 @bernardwang 
> I'm thinking we should use the following format for focus styles

I went ahead and [made this change here](https://github.com/CMSgov/design-system/pull/510/commits/9635e951a46cbad3ba5c1ae3700fcdd5047affdc).

Reply:
@line47 I think these changes have been made, could you update your review?